src/message/NoMessages.js

```diff
@@ -64,7 +65,7 @@ class NoMessages extends PureComponent<Props> {
     return (
       <View style={styles.container}>
         <Label style={styles.text} text={message.text} />
-        <Label text="Why not start the conversation?" />
+        {canSendToNarrow(narrow) ? <Label text="Why not start the conversation?" /> : null}
```

Comment:
I guess
`{canSendToNarrow(narrow) && <Label text="Why not start the conversation?" />}`
would be better.

Reply:
Yeah, I also prefer what you suggest, the only negative is that if we mistakenly are returning a `string` or `number` it tries to render it. I suspect `eslint` should have a rule for that, but I don't think they do.